packages/DropdownMenu/src/Confirmation/Confirmation.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import ContentContainerStyled from "../ContentContainer.styles";
+import ConfirmationStyles from "./Confirmation.styles";
+
+const { func, string } = PropTypes;
+
+const propTypes = {
+  buttonSize: string,
+  confirmButtonType: string,
+  confirmLabel: string.isRequired,
+  description: string,
+  onCancel: func.isRequired,
+  onConfirm: func.isRequired,
+  title: string.isRequired,
+};
+
+const defaultProps = {
+  buttonSize: ShirtSizes.DEFAULT,
+  confirmButtonType: "destructive",
+  description: null,
+};
+
+const Confirmation = ({ title, buttonSize, confirmButtonType, confirmLabel, description, onConfirm, onCancel }) => {
```

Comment:
this is a functional component please use the parameter (prop) and destructured everything in the first line :).

The only place where we destructure props from the parameter is while using style component.

also might be a good idea to rewrite this as:
```js
export default function Confirmation() {
  ... your code 
}
```

^ that will make it clearer that is a component.

Reply:
See also: https://github.com/acl-services/workpapers/pull/11888#discussion_r302283896 @nahumzs